### PR TITLE
Add text firework

### PR DESCRIPTION
**Pull request by smowton**

### CODE REVIEW

File: fireworks/text_firework.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+from lepton import Particle, ParticleGroup, default_system, domain
+from lepton.controller import Lifetime, Movement, Fader, ColorBlender
+from lepton.emitter import StaticEmitter, PerParticleEmitter
+from lepton.renderer import PointRenderer
+from lepton.texturizer import SpriteTexturizer, create_point_texture
+from pyglet import image
+from pyglet.gl import *
+import PIL
+import PIL.ImageDraw
+from window.window import current_y_rotation
```
Comment (thk123) [inline fireworks/text_firework.py]: This looks to be missing?

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 pyglet
 lepton
 discord.py
-python-dotenv
\ No newline at end of file
+python-dotenv
+PIL
```
Comment (thk123) [inline requirements.txt]: I couldn't install this, is this a python2/python3 thing? I used pillow

File: fireworks/text_firework.py (hunk 1 above)
Comment (thk123) [reply fireworks/text_firework.py]: But it doesn't seem to be used so removing it works

**Review by thk123 — CHANGES_REQUESTED** [review]
Are you able to use pillow instead of PIL as the dependency?

**Comment by smowton** [comment]
Changed-- I already had PIL and had just assumed the dependency and module names matched

**Review by thk123 — APPROVED** [review]

**Pull request merged**